lua/ttt2/libraries/bind.lua

```diff
@@ -152,6 +152,7 @@ end
 -- @internal
 local function TTT2BindCheckThink()
 	-- Make sure the user is currently not typing anything, to prevent unwanted execution of a binding.
+	-- TODO: Maybe send vguihandler the pressed button to close menus again or enable fast buys
```

Comment:
We gotta think about vgui frames being open blocking bindings, it's the reason why we cant close roundendscreen with the binding we opened it with. When we setup a quickbuy or similar it's necessary to get keybindings. Maybe some feedthrough for frames, which button was clicked, but generall block the OnPressed-function?

Reply:
what? I'm pretty sure the roundendscreen can be closed by pressing a key

Reply:
> what? I'm pretty sure the roundendscreen can be closed by pressing a key

Just checked: the same key that opens the roundendscreen can't currently close it.